extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -1244,6 +1245,16 @@ angular.module('oppia').component('svgFilenameEditor', {
         if (value === 'fill') {
           ctrl.fillPicker = picker;
         }
+        if (value === 'bg') {
+          ctrl.bgPicker = picker;
+        }
+        picker.onOpen = function() {
+          var alphaSliders = document.querySelectorAll(
+            '.picker_alpha .picker_selector');
+          alphaSliders.forEach(function(element) {
+            element.setAttribute('title', 'Transparency Slider');
+          });
```

Comment:
DOM manipulation in this case is acceptable because this is not configurable in the third-party module. But this needs to be conveyed to devs, otherwise it might get removed in the future e.g. when trying to refactor code. Can you please add a comment for this?

Reply:
Done.